.pipelines/containers/manifest-template.yaml

```diff
@@ -19,9 +19,16 @@ steps:
     retryCountOnTaskFailure: 3
 
   - script: |
-      set -e
+      set -ex
+      echo "checking XDG_RUNTIME_DIR"
+      echo $XDG_RUNTIME_DIR
       make ${{ parameters.name }}-manifest-push
       mkdir -p $(Build.ArtifactStagingDirectory)/images
+
+      echo "setting XDG_RUNTIME_DIR"
+      export XDG_RUNTIME_DIR=/run/user/$(id -u)
+      echo $XDG_RUNTIME_DIR
```

Comment:
Do we want to set this in the task or set this as an env var from the pipeline level?

Reply:
If we set this as an env var in the pipeline then it impacts podman and buildah negatively. This hack is exclusive to skopeo.